android/src/main/java/com/thebluealliance/androidclient/adapters/BindableFragmentPagerAdapter.java

```diff
@@ -0,0 +1,41 @@
+package com.thebluealliance.androidclient.adapters;
+
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.app.Fragment;
+import android.view.ViewGroup;
+
+import com.thebluealliance.androidclient.fragments.DatafeedFragment;
+
+import java.lang.ref.WeakReference;
+import java.util.Hashtable;
+
+/**
+ * A subclass of {@link FragmentPagerAdapter} that stores each fragment in a {@link Hashtable} so
+ * that they can be accessed and bound at any given time.
+ *
+ * Created by Nathan on 8/10/2015.
+ */
+public abstract class BindableFragmentPagerAdapter extends FragmentPagerAdapter implements com.thebluealliance.androidclient.interfaces.BindableFragmentPagerAdapter {
+
+    private Hashtable<Integer, WeakReference<Fragment>> mFragments = new Hashtable<>();
+
+    public BindableFragmentPagerAdapter(FragmentManager fm) {
+        super(fm);
+    }
+
+    @Override
+    public Object instantiateItem(ViewGroup container, int position) {
+        Fragment createdFragment = (Fragment) super.instantiateItem(container, position);
+        mFragments.put(position, new WeakReference<>(createdFragment));
```

Comment:
You should use [getItemId](http://developer.android.com/reference/android/support/v4/app/FragmentPagerAdapter.html#getItemId%28int%29) for the hash in this, because that can take into account other state. Because fragment references can be changed (e.g. with the ViewTeamActivitity dropdown, [see here](https://github.com/the-blue-alliance/the-blue-alliance-android/commit/b531ca11230b2f8c848f1ece7493c04907799a6c#diff-07c460d3108f985e595128b485e5c9eeR60)), there should also be a way to remove them from the map.